test/views/AccessibilityStatementViewSpec.scala

```diff
@@ -0,0 +1,136 @@
+    val dacLink = s"""<a href="${frontendAppConfig.dacUrl}">${messages("accessibilityStatement.howWeTestedThisService.paragraph2.linkText")}</a>"""
+
+    val expectedGuidanceKeys = Seq(
+      messages("accessibilityStatement.title", messages("site.service_name")),
+      messages("accessibilityStatement.heading", messages("site.service_name")),
+      "accessibilityStatement.introduction.paragraph1",
+      messages("accessibilityStatement.introduction.paragraph2", introductionLink),
+      messages("accessibilityStatement.introduction.paragraph3", messages("site.service_name"), ""),
+      "accessibilityStatement.usingThisService.heading",
+      "accessibilityStatement.usingThisService.aboutTheService",
+      "accessibilityStatement.usingThisService.paragraph1",
+      "accessibilityStatement.usingThisService.listItem1",
+      "accessibilityStatement.usingThisService.listItem2",
+      "accessibilityStatement.usingThisService.listItem3",
+      "accessibilityStatement.usingThisService.listItem4",
+      "accessibilityStatement.usingThisService.listItem5",
+      "accessibilityStatement.usingThisService.paragraph2",
+      messages("accessibilityStatement.usingThisService.paragraph3", usingThisServiceLink),
+      "accessibilityStatement.howAccessibleThisServiceIs.heading",
+      messages("accessibilityStatement.howAccessibleThisServiceIs.paragraph1", howAccessibleThisServiceIsLink),
+      "accessibilityStatement.whatToDoIfYouHaveDifficultyUsingThisService.heading",
+      "accessibilityStatement.reportingAccessibilityProblemsWithThisService.heading",
+      messages("accessibilityStatement.reportingAccessibilityProblemsWithThisService.paragraph1", reportingAccessibilityProblemsWithThisServiceLink),
+      "accessibilityStatement.whatToDoIfYouAreNotHappyWithHowWeRespondToYourComplaint.heading",
+      messages("accessibilityStatement.whatToDoIfYouAreNotHappyWithHowWeRespondToYourComplaint.paragraph1", whatToDoIfYouAreNotHappyWithHowWeRespondToYourComplaintLink1, whatToDoIfYouAreNotHappyWithHowWeRespondToYourComplaintLink2),
+      "accessibilityStatement.contactingUsByPhoneOrGettingAVisitFromUsInPerson.heading",
+      "accessibilityStatement.contactingUsByPhoneOrGettingAVisitFromUsInPerson.paragraph1",
+      "accessibilityStatement.contactingUsByPhoneOrGettingAVisitFromUsInPerson.paragraph2",
+      messages("accessibilityStatement.contactingUsByPhoneOrGettingAVisitFromUsInPerson.paragraph3", contactUsLink),
+      "accessibilityStatement.technicalInformationAboutThisServicesAccessibility.heading",
+      "accessibilityStatement.technicalInformationAboutThisServicesAccessibility.paragraph1",
+      messages("accessibilityStatement.technicalInformationAboutThisServicesAccessibility.compliant", howAccessibleThisServiceIsLink),
+      //messages("accessibilityStatement.technicalInformationAboutThisServicesAccessibility.nonCompliant", technicalInformationLink),
```

Comment:
Will these be potentially required later or do they need removing?

Reply:
These are waiting on the DAC review, so have left them in ready